pkg/grizzly/parsing.go

```diff
@@ -9,13 +9,12 @@ import (
 	"io/fs"
 	"os"
 	"path/filepath"
+	"sort"
 	"strings"
 
 	"github.com/google/go-jsonnet"
 	"github.com/grafana/grizzly/pkg/config"
 	"github.com/grafana/tanka/pkg/jsonnet/native"
```

Comment:
After this PR, we should just remove tanka completely as a dependency. Native functions don't do much anymore, all of their features have been introduced into the stdlib

Reply:
Ahh, thanks for that. I tried removing Tanka then hit the nativeFuncs issue. I will look at removing Tanka properly again.